js/components/portal-dashboard/question-overlay.tsx

```diff
@@ -14,6 +14,7 @@ interface IProps {
   currentQuestion?: Map<string, any>;
   currentStudentId: string | null;
   setDashboardViewMode: (mode: DashboardViewMode) => void;
+  setListViewMode: (mode: boolean) => void;
```

Comment:
This name could be improved.  `setListViewMode` implies that it will take a `mode`, but it takes a `boolean`.

Reply:
See my comments below about naming this `toggleQuestionMode` - unless this somehow clashes with upcoming feedback work.

Reply:
Per my comment below, I think the best approach is to change this to:
`setListViewMode: (mode: ListViewMode) => void;`